.gitignore

```diff
@@ -5,4 +5,8 @@
 .DS_Store
 /build
 /captures
-.externalNativeBuild
\ No newline at end of file
+.externalNativeBuild
+.project
+.classpath
+*/bin/
+*/.settings
```

Comment:
Is this change necessary?

Reply:
Eclipse IDE sometimes creates (in my case) folders for in-workspace binaries (/bin) and workspace settings (/.settings) folders per subproject in addition to .classpath and .project files. And gitignore is there to ignore IDE related files/folders.